import_ldraw.py

```diff
@@ -793,7 +863,9 @@ def create_model(self, context, scale):
                     if bpy.ops.object.mode_set.poll():
                         # Go back to object mode, set origin to geometry
                         bpy.ops.object.mode_set(mode='OBJECT')
-                        bpy.ops.object.origin_set(type='ORIGIN_GEOMETRY')
+                        # When using linked bricks, keep original origin point
+                        if not LinkParts:
+                            bpy.ops.object.origin_set(type='ORIGIN_GEOMETRY')
```

Comment:
You are a bit confused by the term 'type' here, I guess. The 'origin_set' command is actually a sort of transformation. You recalculate the origin and you can do this in a few ways. 'ORIGIN_GEOMETRY' is one of them. After recalculating the origin, the type is irrelevant. 

If you would think about something like a default origin-type, I would say this is the 'ORIGIN_CURSOR', but since the cursor does not have a default position, this makes even less sense.

But I'm probably failing to get my point clear. ;-)

Just consider it this way: in the .blend file there is no parameter that defines the type of origin. There is only one type and that is the normal type. The type mentioned above refers to the type of recalculation that is executed.

Hope this makes sense.